apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -306,39 +306,38 @@ def get_query_server_config(name='beeswax', connector=None):
 
 def get_query_server_config_via_connector(connector):
   # TODO: connector is actually a notebook interpreter
-  connector_name = full_connector_name = connector['type']
-  compute_name = None
-  if connector.get('compute'):
-    compute_name = connector['compute']['name']
-    full_connector_name = '%s-%s' % (connector_name, compute_name)
+  compute = connector.get('compute', connector)
```

Comment:
This means we want to assign whole connector object if we dont find compute attribute in connector object?

Reply:
Yes, that is correct. The implementation of connectors/computes is a bit messed up. Sometimes, you get connectors as is while other times it is a compute object inside connectors. It depends on the code-path. This line simplifies a lot of the if-else down below.

Reply:
Ahh I see, make sense to have like this then. Should we add a comment explaining this?